src/neo/Helper.cs

```diff
@@ -310,5 +310,88 @@ internal static BigInteger WeightedAverage<T>(this IEnumerable<T> source, Func<T
                 yield return resultSelector(item, weight);
             }
         }
+
+        public static byte[] Concat(this byte[] a, byte[] b)
```

Comment:
Is it possible to use `Union`?
https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.union?view=netcore-3.1

Reply:
```
        [TestMethod]
        public void TestConcat()
        {
            var a = new byte[] { 0x01, 0x02, 0x03 };
            var b = new byte[] { 0x02, 0x03, 0x04, 0x05 };
            var t1 = DateTime.UtcNow;
            for (int i = 0; i < 100000; i++)
            {
                a.Union(b).ToArray();
                //a.Concat(b);
            }
            var t2 = DateTime.UtcNow;
            Console.WriteLine(t2 - t1);
            Assert.IsTrue(false);
        }
```
I use this ut to compare the time.
When I use `Concat`, it only takes at around 0.0068s. While `Union` takes 0.03s